snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -185,7 +187,7 @@ def initialize_replica(self,
                         f"The override image is not a Postgres image: {incremental_image}")
             except Exception as error:
```

Comment:
should we define a custom exception for "image not available" case?
cc @eculicny

Reply:
That's a year old code, I think better not touch it now